apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -328,7 +344,22 @@ export default function CourseOfferingEditor(props) {
           </select>
         </label>
       ))}
-
+      <h3>Professional Learning</h3>
+      Professional Learning Program
+      <HelpTip>
+        <p>
+          Link to page where they can go to learn more about the professional
+          learning program
+        </p>
+      </HelpTip>
+      <input
+        type="text"
+        value={courseOffering.professional_learning_program}
+        style={styles.input}
+        onChange={e =>
+          updateCourseOffering('professional_learning_program', e.target.value)
```

Comment:
If we only have a few links total for the PL programs, I'm wondering if we'd rather not store this in the database, so that if a path changes, we won't need to worry about updating all of the records.

@dmcavoy Thoughts on this? Do you want this to be editable via the UI?

Reply:
What other way were you thinking about storing it?

Reply:
Hard-coding strings in our repo? There are certainly cons of this approach -- it's less extensible, and impossible for a non-engineer to update. But I worry that if it gets updated via the UI only, it'll be easy to "miss" a URL that needs to be changed, as we'd have to go through every course offering to see. (Unless the person changing it through the UI is Dani, because you'll look in our code too.)

We store workshop paths this way (not saying it's the best!)
https://github.com/code-dot-org/code-dot-org/blob/cc1879fad257cd16bc8e897e94af8c5e73aad0d6/dashboard/lib/pd/workshop_survey_foorm_constants.rb#L26-L35

Reply:
I like this approach. We create constants for the couple of pages (I think its just CSF workshop page and the 6-12 apply page). Then we can just give that constants list as the options to the levelbuilder?

Reply:
That sounds good to me. @juanmanzojr From what Dani shared, it looks like you can (a) keep the column as-is, but (b) ensure it's one of CSF workshop page or 6-12 apply page. You can do this via a [validation](https://guides.rubyonrails.org/active_record_validations.html#acceptance):

https://github.com/code-dot-org/code-dot-org/blob/151f54cea749731032780d403b95be12603a1559/dashboard/app/models/course_offering.rb#L37-L39

or

https://github.com/code-dot-org/code-dot-org/blob/151f54cea749731032780d403b95be12603a1559/dashboard/app/models/regional_partner.rb#L147-L149

I personally like the `validates_inclusion_of` syntax but it might make sense to go with the pattern already in the course_offering model –– it's up to you!